src/models/partial_tx.ts

```diff
@@ -87,17 +62,24 @@ export class ProposalInput extends Input {
   /**
    * Return an object with the relevant input data
    *
-   * @return {InputData}
+   * @return {IDataInput}
    * @memberof ProposalInput
    * @inner
    */
-  toData(): InputData {
-    const data: InputData = {
-      tx_id: this.hash,
+  toData(): IDataInput {
```

Comment:
Not because of this change, since this data is used internally to generate the transaction data, then convert to a `Transaction` instance, the headless itself only sees the `Transaction` instance and the `tx_hex` generated by the `Transaction` instance.
We will require some changes on the headless due to some apis being changed to async though.